installer/dmg/resources/LICENSE.rtf

```diff
@@ -0,0 +1,207 @@
+{\rtf1\ansi\deff0\nouicompat{\fonttbl{\f0\fnil\fcharset0 Courier New;}}

```

Comment:
The formatting of paragraphs are slightly longer than the window that the installer shows so things look misaligned.  It would be nice to have a license that fits better in the window if there is one already formatted to fit.  If not then it's fine to keep as is.

Reply:
![Screen Shot 2023-04-03 at 12 10 45 PM](https://user-images.githubusercontent.com/2888371/229567153-8b32e391-b26d-4225-a1cd-e5b971de248c.png)
It looks normal to me. Does it look different for you?

Reply:
I found a spot that looked wrong and fixed it. Thanks

Reply:
Cool thanks :+1: